### ISSUE TITLE
troubleshooting an empty output

### ISSUE THREAD

**Issue by devonorourke** [agshumate/Liftoff#55]
Good afternoon,
I'd like to be able to lift over the annotations from an older assembly (of a mammalian genome) to my new assembly, and I was hoping the software you've developed would do the trick. While I managed to run the program without error, the resulting .gff file for the target is empty, and the _unmapped.txt_ file contains every gene found in the original assembly. Given that these assemblies came from the same species, I wasn't sure what the first steps in troubleshooting might be.
Thanks for any advice you can offer

**Comment by agshumate** [comment]
Hi,
The first place i would check is in the intermediate_files directory. There should be a fasta file with your gene sequences and a .sam file from the minimap2 output. Check to make sure that the .sam file has gene alignments. If not, try deleting the .mmi and re-running liftoff from the beginning

**Comment by devonorourke** [comment]
Thanks for the quick response @agshumate ,

For my first attempt at this, the job initially ran out of memory, so I ended up just restarting it. I didn't look into which files were created in that first failed attempt after restarting - I just relaunched the script. However, following your advice, it looks like the `.mmi` file was created for the target genome assembly, but was empty:
```
-rw-r--r-- 1 dro49 cluster 438K Dec 18 11:28 mylu_HiC.fasta.fai
-rw-r--r-- 1 dro49 cluster    0 Dec 18 11:28 mylu_HiC.fasta.mmi
-rw-r--r-- 1 dro49 cluster 2.0G Dec 18 11:22 mylu_HiC.fasta
```

I deleted that `.mmi` file, as well as the `.fai` file. I then restarted the same script, and it started to generate the kind of mapping data I was hoping for 😄. I think as far as my initial issue is concerned, your suggestion of deleting the `.mmi` and rerunning was exactly the solution - thanks! 👍 

This sounds a bit silly, but I wonder if it would be of any use to add a flag like `--force` to delete any of those temporary files and restart from the alignment step to avoid this kind of quirk. Glad you solved my issue though - thanks again
